packages/init/__tests__/choose-or-create-repo.test.ts

```diff
@@ -17,12 +17,16 @@ import { createRepository } from "../src/utils/create-repo";
 
 import nock from "nock";
 import { Models } from "@slicemachine/core";
+import { ApplicationMode } from "@slicemachine/client";
+import { Client } from "../src/utils";
 
 import * as fs from "fs";
 
 jest.mock("fs");
 jest.mock("../src/utils/create-repo");
```

Comment:
Because those tests are testing everything that revolves around creating a repo (user input, message displayed, ...) and not the function to create a repo itself.